includes/class-wp-security-bp-files.php

```diff
@@ -230,8 +230,7 @@ public function check_wp_config() {
 			$this->json['message'] = __( 'Good job!!!', $this->plugin_name );
 		}
 
-		wp_send_json( $this->json );
-		wp_die();
+		return json_encode($this->json);
```

Comment:
`wp_send_json()` already does `json_encode()` and, in fact, also `wp_die()` 😝 
https://developer.wordpress.org/reference/functions/wp_send_json/